stub/src/main/java/io/grpc/stub/AbstractStub.java

```diff
@@ -89,10 +89,10 @@ public CallOptions getCallOptions() {
   /**
    * Returns a new stub with the given channel for the provided method configurations.
```

Comment:
channel -> call factory.

There are still "channel" throughout this file. They should all be changed to call factory.